hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferMigrationTest.java

```diff
@@ -27,22 +29,25 @@ public void setup() {
 
     @Test
     public void test() throws Exception {
-        HazelcastInstance hz1 = instanceFactory.newHazelcastInstance();
+        final String ringbufferName = "ringbuffer";
+        final Config config = new Config()
+                .addRingBufferConfig(new RingbufferConfig(ringbufferName).setTimeToLiveSeconds(0));
+        HazelcastInstance hz1 = instanceFactory.newHazelcastInstance(config);
 
         for (int k = 0; k < 10 * CAPACITY; k++) {
-            hz1.getRingbuffer("ringbuffer").add(k);
+            hz1.getRingbuffer(ringbufferName).add(k);
         }
 
-        long oldTailSeq = hz1.getRingbuffer("ringbuffer").tailSequence();
-        long oldHeadSeq = hz1.getRingbuffer("ringbuffer").headSequence();
+        long oldTailSeq = hz1.getRingbuffer(ringbufferName).tailSequence();
+        long oldHeadSeq = hz1.getRingbuffer(ringbufferName).headSequence();
 
-        HazelcastInstance hz2 = instanceFactory.newHazelcastInstance();
-        HazelcastInstance hz3 = instanceFactory.newHazelcastInstance();
+        HazelcastInstance hz2 = instanceFactory.newHazelcastInstance(config);
+        HazelcastInstance hz3 = instanceFactory.newHazelcastInstance(config);
         hz1.shutdown();
 
         sleepSeconds(5);
```

Comment:
Updated the test. The test even lasts less now and it is much clearer. Good suggestion!